From cd5c0d1d4e4b2e5d03935f9b5e10d0dc48ffd3ee Mon Sep 17 00:00:00 2001
From: Wolfgang Merkt <w.merkt@gmail.com>
Date: Thu, 15 Sep 2022 13:19:33 +0100
Subject: [PATCH 1/6] rules.em: Set LDFLAGS=-s

---
 debian/rules.em | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/debian/rules.em b/debian/rules.em
index 284a071..2b70fb7 100755
--- a/debian/rules.em
+++ b/debian/rules.em
@@ -12,7 +12,7 @@ export DH_VERBOSE=1
 # of this sort:
 #  https://code.ros.org/trac/ros/ticket/2977
 #  https://code.ros.org/trac/ros/ticket/3842
-export LDFLAGS=
+export LDFLAGS=-s
 export PKG_CONFIG_PATH=@(InstallationPrefix)/lib/pkgconfig
 # Explicitly enable -DNDEBUG, see:
 # 	https://github.com/ros-infrastructure/bloom/issues/327
-- 
2.25.1

