From 2f29f7229bf52e586622b4e25f79317157fdcd9b Mon Sep 17 00:00:00 2001
From: Guilhem Saurel <guilhem.saurel@laas.fr>
Date: Mon, 11 May 2026 19:42:52 +0200
Subject: [PATCH 1/2] activate collision support

---
 CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 5420dbd216..5b37a38bdd 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -152,7 +152,7 @@ option(
 option(
     BUILD_WITH_COLLISION_SUPPORT
     "Build the library with the collision support (required coal)"
-    OFF
+    ON
 )
 option(
     BUILD_WITH_AUTODIFF_SUPPORT
-- 
2.54.0

