From a4b73b9bb16e1483e3fdf8c27b656c91629a30e4 Mon Sep 17 00:00:00 2001
From: Guilhem Saurel <guilhem.saurel@laas.fr>
Date: Thu, 20 Feb 2025 11:46:18 +0100
Subject: [PATCH 2/3] disable unittests

---
 CMakeLists.txt | 9 ---------
 1 file changed, 9 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 0580726..194f99d 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -452,15 +452,6 @@ add_subdirectory(bindings)
 # --- EXECUTABLES --------------------------------------------------------------
 add_subdirectory(utils)
 
-# --- UNIT TESTS ---------------------------------------------------------------
-add_subdirectory(unittest)
-
-# --- CHECK EXAMPLES -----------------------------------------------------------
-add_subdirectory(examples)
-
-# --- BENCHMARKS ---------------------------------------------------------------
-add_subdirectory(benchmark)
-
 # --- PACKAGING ----------------------------------------------------------------
 macro(EXPORT_VARIABLE var_name var_value)
   get_directory_property(has_parent PARENT_DIRECTORY)
-- 
2.49.0

