From ede4ec3f6c95edfbbd6b7067860afcbe44460482 Mon Sep 17 00:00:00 2001
From: Guilhem Saurel <guilhem.saurel@laas.fr>
Date: Thu, 20 Feb 2025 11:51:13 +0100
Subject: [PATCH 3/4] disable unittests

---
 CMakeLists.txt | 9 ---------
 1 file changed, 9 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 6e75cd4..77321ce 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -460,15 +460,6 @@ add_subdirectory(bindings)
 # --- EXECUTABLES --------------------------------------------------------------
 add_subdirectory(utils)
 
-# --- UNIT TESTS ---------------------------------------------------------------
-add_subdirectory(unittest)
-
-# --- CHECK EXAMPLES -----------------------------------------------------------
-add_subdirectory(examples)
-
-# --- BENCHMARKS ---------------------------------------------------------------
-add_subdirectory(benchmark)
-
 # --- PACKAGING ----------------------------------------------------------------
 macro(EXPORT_VARIABLE var_name var_value)
   get_directory_property(has_parent PARENT_DIRECTORY)
-- 
2.47.0

