From 3599f68b9b5660e37a677389986bdebef78c41de Mon Sep 17 00:00:00 2001
From: Wolfgang Merkt <w.merkt@gmail.com>
Date: Tue, 11 Oct 2022 23:27:10 +0100
Subject: [PATCH 3/6] Override debug level to g1

---
 debian/rules.em | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/debian/rules.em b/debian/rules.em
index 9266b55..3bd7fa9 100755
--- a/debian/rules.em
+++ b/debian/rules.em
@@ -16,7 +16,8 @@ export LDFLAGS=-Wl,-s
 export PKG_CONFIG_PATH=@(InstallationPrefix)/lib/pkgconfig
 # Explicitly enable -DNDEBUG, see:
 # 	https://github.com/ros-infrastructure/bloom/issues/327
-export DEB_CXXFLAGS_MAINT_APPEND=-DNDEBUG
+# Explicitly reduce level of debug symbols to avoid OOM
+export DEB_CXXFLAGS_MAINT_APPEND=-DNDEBUG -g1
 ifneq ($(filter nocheck,$(DEB_BUILD_OPTIONS)),)
 	BUILD_TESTING_ARG=-DBUILD_TESTING=OFF
 endif
-- 
2.47.0

