From ff0d7a7d0d94d5cbca6431e358cf0b0a537b2b55 Mon Sep 17 00:00:00 2001
From: Guilhem Saurel <guilhem.saurel@laas.fr>
Date: Thu, 20 Feb 2025 11:52:11 +0100
Subject: [PATCH 3/3] disable unittests

---
 CMakeLists.txt | 11 -----------
 1 file changed, 11 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 180761f..c659d3c 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -462,17 +462,6 @@ add_subdirectory(bindings)
 # --- EXECUTABLES --------------------------------------------------------------
 add_subdirectory(utils)
 
-# --- UNIT TESTS ---------------------------------------------------------------
-add_subdirectory(unittest)
-
-# --- CHECK EXAMPLES -----------------------------------------------------------
-add_subdirectory(examples)
-
-# --- BENCHMARKS ---------------------------------------------------------------
-if(BUILD_BENCHMARK)
-  add_subdirectory(benchmark)
-endif()
-
 # --- PACKAGING ----------------------------------------------------------------
 macro(EXPORT_VARIABLE var_name var_value)
   get_directory_property(has_parent PARENT_DIRECTORY)
-- 
2.51.0

