From d53c1cddb5d33a6422e055b5273f29ecca63ed47 Mon Sep 17 00:00:00 2001
From: Guilhem Saurel <guilhem.saurel@laas.fr>
Date: Thu, 20 Feb 2025 11:52:11 +0100
Subject: [PATCH 3/3] disable unittests

---
 CMakeLists.txt | 13 -------------
 1 file changed, 13 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index e9540d372..f67ef0af9 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -494,19 +494,6 @@ endif()
 # --- EXECUTABLES --------------------------------------------------------------
 add_subdirectory(utils)
 
-# --- UNIT TESTS ---------------------------------------------------------------
-add_subdirectory(unittest)
-
-# --- CHECK EXAMPLES -----------------------------------------------------------
-if(BUILD_EXAMPLES)
-  add_subdirectory(examples)
-endif()
-
-# --- BENCHMARKS ---------------------------------------------------------------
-if(BUILD_BENCHMARK AND NOT BUILD_STANDALONE_PYTHON_INTERFACE)
-  add_subdirectory(benchmark)
-endif()
-
 # --- PACKAGING ----------------------------------------------------------------
 macro(EXPORT_VARIABLE var_name var_value)
   get_directory_property(has_parent PARENT_DIRECTORY)
-- 
2.52.0

