From 1f99d34bc52b2326bcccd49d0a7569a18d2261b7 Mon Sep 17 00:00:00 2001
From: Guilhem Saurel <guilhem.saurel@laas.fr>
Date: Sat, 22 Feb 2025 11:32:45 +0100
Subject: [PATCH 4/4] CMake: fix for hpp-fcl/coal v3

---
 CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 77321ce..465aab5 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -338,7 +338,7 @@ else()
 endif()
 
 if(BUILD_WITH_HPP_FCL_SUPPORT)
-  add_project_dependency(hpp-fcl 2.1.2 REQUIRED "hpp-fcl >= 2.1.2")
+  add_project_dependency(hpp-fcl REQUIRED)
 endif()
 
 if(BUILD_WITH_ACCELERATE_SUPPORT)
-- 
2.47.0

