From 271420ff67a4cea07d6d4b8f422c4a272defca4e Mon Sep 17 00:00:00 2001
From: Wolfgang Merkt <w.merkt@gmail.com>
Date: Mon, 30 Jan 2023 06:54:32 +0000
Subject: [PATCH 4/6] Remove -Wl,-s LDFLAGS

---
 debian/rules.em | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/debian/rules.em b/debian/rules.em
index 3bd7fa9..a7f8836 100755
--- a/debian/rules.em
+++ b/debian/rules.em
@@ -12,7 +12,7 @@ export DH_VERBOSE=1
 # of this sort:
 #  https://code.ros.org/trac/ros/ticket/2977
 #  https://code.ros.org/trac/ros/ticket/3842
-export LDFLAGS=-Wl,-s
+export LDFLAGS=
 export PKG_CONFIG_PATH=@(InstallationPrefix)/lib/pkgconfig
 # Explicitly enable -DNDEBUG, see:
 # 	https://github.com/ros-infrastructure/bloom/issues/327
-- 
2.43.0

