From 27cbaf67819ac1e7176434f0b9244eb042658bc0 Mon Sep 17 00:00:00 2001
From: Wolfgang Merkt <w.merkt@gmail.com>
Date: Wed, 1 Feb 2023 00:17:00 +0000
Subject: [PATCH 6/6] Adding -fno-lto to CXXFLAGS as well

---
 debian/rules.em | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/debian/rules.em b/debian/rules.em
index 746e698..26af70b 100755
--- a/debian/rules.em
+++ b/debian/rules.em
@@ -17,7 +17,7 @@ export PKG_CONFIG_PATH=@(InstallationPrefix)/lib/pkgconfig
 # Explicitly enable -DNDEBUG, see:
 # 	https://github.com/ros-infrastructure/bloom/issues/327
 # Explicitly reduce level of debug symbols to avoid OOM
-export DEB_CXXFLAGS_MAINT_APPEND=-DNDEBUG -g1
+export DEB_CXXFLAGS_MAINT_APPEND=-DNDEBUG -g1 -fno-lto
 ifneq ($(filter nocheck,$(DEB_BUILD_OPTIONS)),)
 	BUILD_TESTING_ARG=-DBUILD_TESTING=OFF
 endif
-- 
2.47.0

